src/docs/usage/validateCartItems().mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: validateCartItems
+---
+
+`validateCartItems(inventory, cartItems)`
+
+validateCartItems checks the items in the current cart and compares them with a source of truth to
+make sure that the prices and quantities are correct. This is necessary because it's possible that
+malicious users could manipulate the values of prices in the DOM before sending to `redirectToCheckout`
+
+inventory: Source of truth. This should be the object of products you sell. It can come from anywhere that
+is secure. An API call, or a json file that sits on a server.
+
+cartItems: the cartItems that comes from the `use-shopping-cart`
+
+Take a look at the Netlify example here:
```

Comment:
The Netlify example is good but we need a minimal reproduction of how to use it from both a server (or serverless function) and the front-end in the documentation

Reply:
For this, I was thinking of creating an `examples` category where we will be having similar examples done with CRA, Gatsby, Next etc etc

What do you think?

Reply:
@dayhaysoos that would be cool, however, for this page we should be able to also provide a simplistic view of how it should be used in a server and front-end environment. Essentially a code example that explains the overall idea of how to use `validateCartItems` and of course we can still provide in-depth frameworks specific examples.

The front-end part of the example should essentially remain the same between examples though I believe.

Reply:
@ChrisBrownie55  Cool

One more thing that I thought of, maybe we can save this for a later upgrade to the docs.

Hooks work with mdx, so we could actually wrap the docs in CartProvider and provide live, interactive examples on the docs themselves.

This would be a good case to start working on the <DebugCart /> component you talked about earlier

Reply:
That'd be nice, I've seen a lot of other docs with MDX setups where you can view both the code and the live example.

If it turns out that this isn't feasible it should be possible to do CodeSandbox embeds.

We can also use Glitch embeds for examples that also encompass the back-end.